### ISSUE TITLE
RestServer code on reference page does not match the library's example sketch

### ISSUE THREAD

**Issue by per1234** [arduino-libraries/Ciao#2]
The code at https://www.arduino.cc/en/Reference/CiaoRestServer does not match https://github.com/arduino-libraries/Ciao/blob/master/examples/RestServer/RestServer.ino and the code on the reference page has multiple errors:

- Many instances of `elseif` used instead of `else if`
- `splitString()` used instead of `Ciao.splitString()`

Originally reported at: http://forum.arduino.cc/index.php?topic=513519

**Comment by novski** [comment]
still open. how come?

**Comment by q2dg** [comment]
This library is for retired products (https://www.arduino.cc/en/Reference/Ciao) so...it seems pretty abandoned

**Comment by MisterAwesome23** [comment]
Looks like this document (the one on Github) is the updated one.
Minor changes, 
`String sender` which is not actually needed is removed.
`String message` has been replaced with `String request`

It does not have any impact on the functioning of the program. Just for maintaining a standard code throughout, should I make a pull with the code present on reference page?

**Comment by per1234** [comment]
@MisterAwesome23 no. The code on the reference page is seriously flawed. Instead, the code from this repository needs to be copied to the reference page. Only someone with edit access to the documentation content can do that.

**Comment by MisterAwesome23** [comment]
> The code on the reference page is seriously flawed.

I agree. The one here on GitHub is the updated one. Though I'm still not sure on the seriously flawed part. 
I see the ```elseif``` has to change to ```else if``` certainly. But other than this, just comparing the two, the updated one here on github removes a redundancy (variable sender) and makes appropriate variable name change from ```message``` to ```request```.

> Only someone with edit access to the documentation content can do that.

True. As mentioned on the reference page- 
"Corrections, suggestions, and new documentation should be posted to the Forum." 
I'll just post this code there so that someone there can simply copy paste than needing to navigate here.

Link to forum post- https://forum.arduino.cc/index.php?topic=678083.0

Thanks.
